src/DataObjects/WorkflowDefinition.php

```diff
@@ -482,12 +482,7 @@ public function incomingTitle()
      */
     public function canWorkflowPublish($member, $target)
     {
-        $publish = $this->extendedCan('canWorkflowPublish', $member, $target);
-
-        if (is_null($publish)) {
-            return false;
-        }
-        return $publish;
+        return $this->extendedCan('canWorkflowPublish', $member, $target);
```

Comment:
If no extensions modify this permission check, it will default to null. Unfortunately, `(bool)null === false` so the can check might still incorrectly evaluate to false.